apps/mymedicare_cb/authorization.py

```diff
@@ -280,14 +280,25 @@ def service_health_check(self, request):
         the BB2 /health/external check.
         """
         headers = self.slsx_common_headers(request)
-
-        response = requests.get(
-            self.healthcheck_endpoint,
-            headers=headers,
-            allow_redirects=False,
-            verify=self.verify_ssl_internal,
-            timeout=5,
-        )
+        max_retries = 3
+        retries = 0
+        while retries < max_retries:
+            try:
+                response = requests.get(
+                    self.healthcheck_endpoint,
+                    headers=headers,
+                    allow_redirects=False,
+                    verify=self.verify_ssl_internal,
+                    timeout=10,
+                )
+                response.raise_for_status()
+                return True
+            except requests.exceptions.RequestException:
+                if retries < max_retries:
+                    print(f"Request failed. Retrying... ({retries+1}/{max_retries})")
+                    retries += 1
+                else:
+                    return False
```

Comment:
There's some question of what the ideal behavior is in cases where the 3 retries all fail, to more closely match what we have today, we can re-raise the exception by doing something like the suggestion below. Otherwise, I'm not sure what the observed behavior will be if we return `false`. While we're at it, if we want to include `e` in the printed error/retry message, we can do that, although I'm not sure if there's anything useful there or not.
```suggestion
            except requests.exceptions.RequestException as e:
                if retries < max_retries:
                    print(f"Request failed. Retrying... ({retries+1}/{max_retries})")
                    retries += 1
                else:
                    raise e
```